core-api/src/test/java/com/optimizely/ab/bucketing/BucketerTest.java

```diff
@@ -79,6 +70,7 @@ public void generateBucketValueForNegativeHashCodes() throws Exception {
     @Test
     @Category(ExhaustiveTest.class)
     public void generateBucketValueDistribution() throws Exception {
+        Assume.assumeTrue(System.getProperty("os.name").contains("Linux"));
```

Comment:
http://docs.oracle.com/javase/6/docs/api/java/lang/System.html#getenv(java.lang.String)

 we could use an environment variable `RUN_EXHAUSTIVE_BUCKETING`